saos-restful-web-api/src/main/java/pl/edu/icm/saos/api/dump/enrichmenttag/DumpEnrichmentTagsListSuccessRepresentationBuilder.java

```diff
@@ -96,7 +99,7 @@ private QueryTemplate toQueryTemplate(Pagination pagination) {
         QueryTemplate queryTemplate = new QueryTemplate();
 
         queryTemplate.setPageNumber(new PageNumberTemplate(pagination.getPageNumber()));
-        queryTemplate.setPageSize(new PageSizeTemplate(pagination.getPageSize()));
+        queryTemplate.setPageSize(new PageSizeTemplate(pagination.getPageSize(), parametersExtractor.getMinPageSize(), parametersExtractor.getMaxPageSize()));
```

Comment:
Maybe we should have a sort of a QueryTemplateFactory to avoid duplication of these lines in many places.